lang/en.json

```diff
@@ -204,11 +204,12 @@
         "ExceptionInvalidRendererFormat": "Renderer format '%1$s' not valid. Try any of the following instead: %2$s.",
         "ExceptionInvalidReportRendererFormat": "Report format '%1$s' not valid. Try any of the following instead: %2$s.",
         "ExceptionInvalidStaticGraphType": "Static graph type '%1$s' not valid. Try any of the following instead: %2$s.",
-        "ExceptionInvalidToken": "Token is not valid.",
+        "ExceptionInvalidTokenNew": "Token is not valid or expired, please refresh the page and try again.",
```

Comment:
Actually I think that change doesn't make much sense. The message is shown if the `token_auth` provided in the request, does not match the `token_auth` of the current user. If that is the case, refreshing the page won't help anything, as the token in the request would still be the same and therefor be invalid.

Reply:
@sgiehl Does saying "logout and login again to resolve the issue" would be fine ?

Reply:
Had some more thoughts around that: I guess this method is mostly used in actions that are called in ajax requests. So this message might be shown as a notification if the ajax request fails (due to an incorrect token).
I'm actually wondering if the message "Token is not valid" is helpful for anyone at all. We should maybe try to use less technical error message so the user is able to understand the problem.
In that case maybe something like this might work:
`An authentication error occurred, please refresh the page and try again.`
@tsteur @mattab what's your opinion regarding error messages?

Reply:
Great thinking. We could adjust it. I'm not sure "authentication" is quite the right word as it could be misleading also. What about something like this?

"We're sorry, but the action you were trying to perform failed due to a security check. This is a standard measure to ensure the safety and security of your account. Please try again, and if the issue persists, please contact our support team for further assistance."

Reply:
Yes. That one sounds nice. Maybe we could even use the same message for both errors. For the user it doesn't make a difference if it was an authentication token or a nonce that doesn't match.
@AltamashShaikh Could you update the PR accordingly?